packages/UI/src/components/Checkbox/Checkbox.tsx

```diff
@@ -58,7 +58,9 @@ const checkboxStyles = tv({
  */
 
 export default function Checkbox({ className, ...props }: CheckboxProps) {
-  const { rootStyle, indicatorStyle } = checkboxStyles();
+  const { rootStyle, indicatorStyle } = checkboxStyles({
+    className,
```

Comment:
when someone passes a className prop, which element/slot would be affected here?

Reply:
Oh both, fixing it now.